include/aspect/simulator.h

```diff
@@ -1911,6 +1914,8 @@ namespace aspect
       friend class MeshDeformation::MeshDeformationHandler<dim>;   // MeshDeformationHandler needs access to the internals of the Simulator
       friend class VolumeOfFluidHandler<dim>; // VolumeOfFluidHandler needs access to the internals of the Simulator
       friend class StokesMatrixFreeHandler<dim>;
+      template <int dimi, int velocity_degree>
```

Comment:
```suggestion
      template <int dim, int velocity_degree>
```

Reply:
that doesn't work:
``
../include/aspect/simulator.h:145:13: error:  shadows template parm ‘int dim’
``

Reply:
but I changed it to ``dimension``

Reply:
Ok then. Do you think the current state is ready?